### PR TITLE
[DEN-296] Update Readme.MD for bot rules

### PR DESCRIPTION
**Pull request by stevenpaz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -279,7 +279,38 @@ To use the WAF service, use the API Token provided to you.
 An access rule identifies legitimate traffic and threats by ASN, Cookie, Country, 
 IP Address, Referrer, URL, User agent, HTTP method, Media type, File extension, 
 and Request headers.
-```
+
+For detailed information about Access Rules in WAF, please read the [official documentation](https://docs.edgecast.com/cdn/#Web-Security/Access-Rules.htm).
+
+#### Bot Rules
+Text TBD
```
Comment (wdawg33) [inline README.md:286]: We just need a ticket (or tie this to the SDK ticket) to update once we have the documentation from Jose.

**Review by wdawg33 — APPROVED** [review]

**Review by defstream — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (stevenpaz) [reply README.md]: Updated

File: README.md [hunk 2]
```diff
@@ -279,7 +279,45 @@ To use the WAF service, use the API Token provided to you.
...
 and Request headers.
-```
+
+For detailed information about Access Rules in WAF, please read the [official documentation](https://docs.edgecast.com/cdn/#Web-Security/Access-Rules.htm).
+
+#### Bot Rule Sets
+Use bot rules to require a client (e.g., a web browser) to solve a challenge 
+before resolving the request. WAF blocks traffic when the client cannot solve 
+this challenge within a few seconds. Basic bots typically cannot solve this type
+of challenge and therefore their traffic is blocked. This prevents them from
```
Comment (defstream) [inline README.md:289]: Basic bots typically cannot solve this type
of challenge, and therefore their traffic is blocked.

File: README.md [hunk 3]
```diff
@@ -279,7 +279,45 @@ To use the WAF service, use the API Token provided to you.
...
+of challenge and therefore their traffic is blocked. This prevents them from 
+scraping your site, carding, spamming your forms, launching DDoS attacks, and 
+committing ad fraud.
+
+For detailed information about Bot Rules in WAF, please read the [official documentation](https://docs.edgecast.com/cdn/#Web-Security/Bot-Rules.htm).
+
+#### Custom Rule Sets
+Use custom rules to tailor how WAF identifies malicious traffic. This provides
+added flexibility for threat identification that allows you to target malicious
+traffic with minimal impact to legitimate traffic. Custom threat identification
```
Comment (defstream) [inline README.md:298]: This provides
added flexibility for threat identification that allows you to target malicious
traffic with minimal impact on legitimate traffic.

**Review by defstream — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (stevenpaz) [reply README.md]: Updated

File: README.md (hunk 3 above)
Comment (stevenpaz) [reply README.md]: Updated

**Comment by stevenpaz** [comment]
@defstream updated grammar changes

**Review by defstream — APPROVED** [review]

**Pull request merged**